src/test/java/jchess/ruleengine/DefaultRuleProcessorTest2_2PlayerBoard.java

```diff
@@ -57,7 +59,7 @@ static void tearDownAfterClass() throws Exception {
 	void setUp() throws Exception {
 		IAppLogger oLogger = new AppLogger();
 		ICacheManager oCacheManager = new CacheManager(oLogger);
-		
+		m_oRuleEngine = new DefaultRuleEngine(new DefaultRuleProcessor(oLogger), new GUIManagerTest(oLogger), oLogger);
```

Comment:
This is not right. If you need to test DefaultRuleEngine then move your test to DefaultRuleEngineTest class.